src/schema/schema.ts

```diff
@@ -199,41 +197,6 @@ export function createZimbraSchema(
 					return obj.conversationId ? 'MessageInfo' : 'Conversation';
 				}
 			},
-			Folder: {
```

Comment:
Why did we remove this?

Reply:
Found no usage of it.

Reply:
@SantoshKShetty should we remove the respective `folder -> appointments` mapping from schema.graphql too?

Reply:
Nice catch. Fixed.

Reply:
For now, I have removed only `folder -> tasks` mapping from schema.graphql as we're still using CalendarsAndAppointmentsQuery in new-event-from-email zimlet.

Once we refactor that zimlet, we'll update schema.graphql and remove `folder -> appointments`